packages/form/src/components/KeyValueField/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { Button, Row, Stack } from '@ultraviolet/ui'
+import type { ComponentProps } from 'react'
+import { useFieldArray } from 'react-hook-form'
+import { TextInputField } from '../TextInputField'
+
+type InputProps = {
+  label: ComponentProps<typeof TextInputField>['label']
+  required?: ComponentProps<typeof TextInputField>['required']
+  regex?: ComponentProps<typeof TextInputField>['regex']
+}
+
+type InputValueProps = {
+  type?: ComponentProps<typeof TextInputField>['type']
+  placeholder?: ComponentProps<typeof TextInputField>['placeholder']
+} & InputProps
+
+type AddButtonProps = {
+  name: ComponentProps<typeof Button>['children']
+  fullWidth?: ComponentProps<typeof Button>['fullWidth']
+  tooltip: string
+  tooltipAlert: string
```

Comment:
Took me a bit of time to understand this. Do we really need tooltip? Is there any case where we have the button that is not disabled and with a tooltip? If not I think we can only keep tooltipAlert.

For tooltipAlert I will also add that it's not really clear what it does so two possibilites:
1. We rename it to make it more explicit
2. We explain each props in the story or we will forget the use of this prop